l1-contracts/src/core/Rollup.sol

```diff
@@ -94,5 +96,17 @@ contract Rollup is IRollup {
     if (version != VERSION) {
       revert Errors.Rollup__InvalidVersion(version, VERSION);
     }
+
+    if (ts > block.timestamp) {
+      revert Errors.Rollup__TimestampInFuture();
+    }
+
+    // @todo @LHerskind consider if this is too strict
```

Comment:
is there a relevant issue for us to come back to this later or is the plan to hash this out now?